inc/mlx90632.h

```diff
@@ -324,6 +363,30 @@ void mlx90632_set_emissivity(double value);
  */
 double mlx90632_get_emissivity(void);
 
+/** Trigger start of burst measurement for mlx90632
+ *
+ * Trigger a single measurement cycle and wait for data to be ready. It does not read anything, just triggers and completes.
+ * The SOB bit is set so that the complete measurement table is re-freshed.
+ * @note The SOB bit is cleared internally by the mlx90632 immediately after the measurment has started.
+ *
+ * @retval <0 Something failed. Check errno.h for more information
+ * @retval 0 New data is available and waiting to be processed
+ *
+ * @note This function is using usleep so it is blocking!
+ */
+int32_t mlx90632_start_measurement_burst(void);
+
+/** mlx90632 i2c addressed reset
+ *
+ * Send a reset command through i2c to a mlx90632 device with a specified slave address.
+ * It also waits for at least 150us to ensure the mlx90632 device is properly reset and ready for further communications.
+ *
+ * @retval <0 Something failed. Check errno.h for more information
+ * @retval =0 The mlx90632 device is propely reset and ready for communication.
```

Comment:
Retval implies equal. Not needed explicitly as well.

```suggestion
 * @retval 0 The mlx90632 device is propely reset and ready for communication.
```